pkg/storage/client_raft_test.go

```diff
@@ -597,6 +597,59 @@ func TestReplicateAfterTruncation(t *testing.T) {
 	})
 }
 
+func TestRaftLogSizeAfterTruncation(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	mtc := startMultiTestContext(t, 3)
+	defer mtc.Stop()
+
+	const rangeID = 1
+	mtc.replicateRange(rangeID, 1, 2)
+
+	repl, err := mtc.stores[0].GetReplica(rangeID)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	incArgs := incrementArgs([]byte("a"), 5)
+	if _, err := client.SendWrapped(
+		context.Background(), rg1(mtc.stores[0]), &incArgs); err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
I think you should probably add in:

```
	mtc.waitForValues([]byte("a"), []int64{5, 5, 5})
```

Or this could have problems with stress.